SPEC.md

```diff
@@ -4330,7 +4346,7 @@ Example input:
 
 ```json
 {
-  "bash_variables.str": "hello"
```

Comment:
Should we not keep this? Seems prudent to keep it for execution engines that will require the prefixed key.

Reply:
Agreed. This is when i was doing some testing.